os/lib/assert.c

```diff
@@ -27,15 +27,16 @@
  * SUCH DAMAGE. 
  *
  */
+#include "lib/assert.h"
 
 #include <stdio.h>
 
 void
 _xassert(const char *file, int lineno)
 {
   printf("Assertion failed: file %s, line %d.\n", file, lineno);
-  /*
-   * loop for a while;
-   * call _reset_vector__();
-   */
+
+#if !ASSERT_RETURNS
+  while(1);
```

Comment:
It would be nicer to users to print something before entering the while loop, like "The program (firmware) stops running."

Reply:
Something like that?

Reply:
Looks good. Thanks!